lib/sqlalchemy/sql/schema.py

```diff
@@ -821,6 +826,9 @@ def listen_for_reflect(table, column_info):
             _implicit_generated=True
         )._set_parent_with_dispatch(self)
         self.foreign_keys = set()  # type: ignore
+        self.periods = set()
+        self.system_versioning = False
```

Comment:
We probably can get away with iterating through "self.periods" at compilation time and compiling on those explicitly, that would render the table suffixes, and not have actual self.system_versioning / self.application_versioning attributes.   if at all possible we want to try to not hold onto the same information in multiple ways.

Reply:
This was refactored, SV information is now only contained as an element in Table.periods